apps/web/components/templates/challenges/all/allChallenges.tsx

```diff
@@ -0,0 +1,44 @@
+import { ChallengeItem } from 'molecules/challengeItem/challengeItem';
+
+type challenge = {
+	title: string;
+	description: string;
+	designs: string;
+	difficulty: string;
+	image: string;
+};
+
+const challenges: challenge[] = [
+	{
+		title: 'First Challenge',
+		description:
+			'This is example of challenge. In hac habitasse platea dictumst. Praesent eu auctor velit. Cras mattis gravida odio euismod placerat. Morbi vestibulum dapibus diam, a lacinia felis porttitor vel. Ut sodales tincidunt orci, vel condimentum nibh. Fusce in elit euismod, laoreet felis vel, rutrum lacus.',
+		designs: '',
+		difficulty: 'medium',
+		image:
+			'https://images.unsplash.com/photo-1498008122250-bcb854c8462d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2070&q=80',
+	},
+	{
+		title: 'Second Challenge',
+		description:
+			'This is example of challenge. In hac habitasse platea dictumst. Praesent eu auctor velit. Cras mattis gravida odio euismod placerat. Morbi vestibulum dapibus diam, a lacinia felis porttitor vel. Ut sodales tincidunt orci, vel condimentum nibh. Fusce in elit euismod, laoreet felis vel, rutrum lacus.',
+
+		designs: '',
+		difficulty: 'advanced',
+		image:
+			'https://cdn.pixabay.com/photo/2020/11/28/06/15/cold-5783718_1280.jpg',
+	},
+];
+
+export const AllChallenges = () => {
+	return (
+		<>
+			<div className="mx-auto w-11/12">
+				<h1>Strona z zadaniami</h1>
+				{challenges.map((challenge) => (
+					<ChallengeItem challenge={challenge} />
+				))}
+			</div>
+		</>
```

Comment:
Not needed wrapping `React.Fragment`, there is a top-level div 😊 

And speaking about a11y, what do you think about using a div here? Challenges seem to be a meaningful container. Maybe a `section` tag will be better here? 
And if you want to be really a11y strict it's nice to pass an attribute `aria-labelledby` to a heading and associate it with a `section`

Reply:
Yes, in fact `React.Fragment` is unnecessary - he had to stay like I tested something.

You think about one big section with list of all challenges like this:
```tsx
<section className='className="mx-auto w-11/12' aria-labelledby="pageHeading">
	<h1 id="pageHeading">Strona z zadaniami</h1>
	{challenges.map((challenge) => (
		<ChallengeItem challenge={challenge} />
	))}
</section>
```
or about every challenge in section tag?

Reply:
Yes, this snippet of the code that you sent is fine. 
And when I've taken a look at this once again I realised that this could be done with ul and li
```ts
<section className='className="mx-auto w-11/12' aria-labelledby="pageHeading">
  <h1 id="pageHeading">Strona z zadaniami</h1>
  <ul>
    {challenges.map((challenge) => (
      <ChallengeItem challenge={challenge} />
    ))}
  </ul>
</section>
```
And every `ChallengeItem` will be an `li` element. What do you think about it?

Reply:
Yes, it's good idea.